pkg/controller/tuned/tuned_controller.go

```diff
@@ -122,6 +179,20 @@ func (r *ReconcileTuned) syncServiceAccount(tuned *tunedv1.Tuned) error {
 	} else {
 		// Do not update service account as it leads to accumulating secrets; see BZ1714484
 		glog.V(2).Infof("Tuned ServiceAccount already exists, skipping update")
+
+		isUpgrade, err := r.isOperandUpgrade(tuned)
+		if err != nil {
+			glog.Errorf("%v", err)
+			return nil
```

Comment:
Probably want to `return err`

Reply:
I'd rather not clean than re-trigger the reconciliation loop.